wifiphisher/common/phishinghttp.py

```diff
@@ -20,6 +20,24 @@ def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
+class ValidateHandler(tornado.web.RequestHandler):
+
+    def initialize(self, em):
+        self.em = em
+
+    def get(self, cred_type, cred_content):
+        """
+        Override the get method
+
+        :param self: A tornado.web.RequestHandler object
+        :type self: tornado.web.RequestHandler
+        :return: None
+        :rtype: None
+        """
+        value = self.em.verify_cred((cred_type, cred_content))
+        self.write("%s" % value)
+
```

Comment:
@sophron, thanks for the guidence. This PR looks good for me.

I have written simple GET request for URI: `/validate/psk/passphrase` from the file: `index.html` in the `firmware_upgrade` phishing scenario and it works properly.